README.md

```diff
@@ -15,6 +15,9 @@
 	</a>
 	<a href="https://gitter.im/typeorm/typeorm?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge">

```

Comment:
@pleerock Should we keep both chats or are we migrating fully to slack and gitter should be deprecated. Merging #2095 depends on that decision.

Reply:
@pleerock ? I would vote for slack only.

Reply:
https://github.com/typeorm/typeorm/pull/3207 @pleerock